### PR TITLE
hectormata.md profile

### PR DESCRIPTION
**Pull request by hectormata**
[rawgit link]
https://rawgit.com/hectormata/hectormata.github.io/First_branch/#!pages/vi/profiles/hectormata.md

### CODE REVIEW

**Comment by hectormata** [comment]
[rawgit link]
https://rawgit.com/hectormata/hectormata.github.io/First_branch/#!pages/profiles/vi/hectormata.md

**Comment by thesiege208** [comment]
@hectormata Your link doesn't work; it should be `vi/profiles`, not `profiles/vi`. Otherwise, it looks good! :)

**Comment by hectormata** [comment]
@thesiege208 Thank you, Just fixed the link!

File: pages/vi/profiles/hectormata.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+* OS: MacOS High Sierra | version 10.13.2
+
+
+I attend the University of Massachussetts in Boston, currently studying computer science with an anticipated graduation day May, 2018. 
+I have a passion for technology because it helps our society be better, learn more, and achieve more as a race. I enjoy programming, my favorite
+programming language is Java. Besides programming, I really enjoy playing/watching **soccer**. My favorite team is Barcelona F.C
+
+Programming Skills:
+****
```
Comment (yaoxing1990) [inline pages/vi/profiles/hectormata.md]: @hectormata Interesting new element

**Pull request merged**